biz/main/src/main/kotlin/io/goooler/demoapp/main/bean/MainRepoListBean.kt

```diff
@@ -12,14 +12,14 @@ import com.squareup.moshi.JsonClass
 @Entity(tableName = "main_repo_list")
 class MainRepoListBean(
   @PrimaryKey val id: Long,
-  @Ignore
-  val private: Boolean,
   val name: String,
   @Json(name = "full_name")
   @ColumnInfo(name = "full_name")
   val fullName: String,
   @Embedded val owner: OwnerBean,
 ) {
+  @Ignore
+  var private: Boolean = false
```

Comment:
```
Entities and POJOs must have a usable public constructor. 
You can have an empty constructor or a constructor whose parameters match the fields (by name and type).
Tried the following constructors but they failed to match:
```

Reply:
We can't annotate fields with `@Ignore` in primary constructor, this is a bug of Room, see https://issuetracker.google.com/issues/70762008